frontend/src/models/Pose.ts

```diff
@@ -1,19 +1,10 @@
-export interface Position {
-    x: number
-    y: number
-    z: number
-}
-
-export interface Orientation {
-    x: number
-    y: number
-    z: number
-    w: number
-}
+import { Orientation } from './Orientation'
+import { Position } from './Position'
 
 export interface Pose {
     position: Position
     orientation: Orientation
+    frame?: string
```

Comment:
Do we need the frame or could we just remove it? Could it be anything other than asset?

Reply:
I'd argue that we should remove it both in the frontend and backend unless we want Flotilla to use it actively. We can argue that all Poses in Flotilla is by default asset as we cannot handle other than asset

Reply:
True